ui/app/AppLayouts/Wallet/views/TransactionDetailView.qml

```diff
@@ -543,16 +542,16 @@ Item {
                     TransactionDataTile {
                         width: parent.width
                         title: !!transactionHeader.networkName ? qsTr("Included in Block on %1").arg(transactionHeader.networkName) : qsTr("Included on Block")
-                        subTitle: d.blockNumber
+                        subTitle: d.blockNumberOut
                         tertiaryTitle: root.isTransactionValid ? LocaleUtils.formatDateTime(transaction.timestamp * 1000, Locale.LongFormat) : ""
-                        visible: d.blockNumber > 0
+                        visible: d.blockNumberOut > 0
                     }
                     TransactionDataTile {
                         width: parent.width
-                        title: !!d.toNetworkName ? qsTr("Included in Block on %1").arg(d.toNetworkName) : qsTr("Included on Block")
-                        subTitle: d.toBlockNumber
+                        title: !!transactionHeader.networkNameOut ? qsTr("Included in Block on %1").arg(transactionHeader.networkNameOut) : qsTr("Included on Block")
```

Comment:
Mixing Out and In data here, probably the reason for what I shared in the screenshot